include/fuzzuf/algorithms/afl/afl_mutation_hierarflow_routines.hpp

```diff
@@ -28,8 +28,6 @@
 #include "fuzzuf/hierarflow/hierarflow_routine.hpp"
 #include "fuzzuf/optimizer/optimizer.hpp"
 
-#include "fuzzuf/optimizer/optimizer.hpp"
```

Comment:
I just accidentally found that this header is included twice. So I just removed it.